tests/cases/fourslash/organizeImportsType8.ts

```diff
@@ -20,33 +20,31 @@ console.log(a, b, A, B);`,
 
 edit.replaceLine(0, 'import { type A, type a, b, B } from "foo2";');
 verify.organizeImports(
-`import { B, b, type A, type a } from "foo2";
+`import { b, B, type A, type a } from "foo2";

```

Comment:
I was going to ask how this was determining `b, B` instead of `B, b`, but I guess it's a 50/50 split between the two given it's `A, a, b, B`?

Reply:
The original ordering is `b, B`, and it should preserve that since it's still ordered case-insensitively